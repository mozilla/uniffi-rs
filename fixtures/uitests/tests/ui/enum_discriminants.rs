fn main() {} /* empty main required by `trybuild` */

#[repr(u64)]
#[derive(uniffi::Enum)]
pub enum EnumExpr {
    V = 1 + 1,
}

uniffi_macros::setup_scaffolding!();
