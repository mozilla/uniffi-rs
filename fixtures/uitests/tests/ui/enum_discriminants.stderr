error: UniFFI disciminant values must be a literal
 --> tests/ui/enum_discriminants.rs:6:9
  |
6 |     V = 1 + 1,
  |         ^^^^^
